src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterArgumentCollection.java

```diff
@@ -0,0 +1,41 @@
+package org.broadinstitute.hellbender.cmdline.GATKPlugin;
+
+import java.io.Serializable;
+import java.util.Collection;
+import java.util.Collections;
+
+/**
+ * An abstract ArgumentCollection for defining the set of read filter descriptor plugin arguments that are exposed to the user on the command line.
+ *
+ * Subclasses should provide {@link org.broadinstitute.barclay.argparser.Argument} annotations for the arguments that should be exposed.
+ *
+ * @author Daniel Gomez-Sanchez (magicDGS)
+ * @see org.broadinstitute.hellbender.cmdline.GATKPlugin.GATKReadFilterPluginDescriptor
+ */
+public abstract class GATKReadFilterArgumentCollection implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    /**
+     * Returns the enabled filter names. Order should be honored.
+     *
+     * Default implementation returns an empty list.
+     */
+    public Collection<String> getUserEnabledReadFilterNames() {
+        return Collections.emptyList();
+    }
+
+    /**
+     * Returns the disabled read filter names. Order should be honored.
+     *
+     * Default implementation returns an empty list.
+     */
+    public Collection<String> getUserDisabledReadFilterNames() {
+        return Collections.emptyList();
+    }
+
+    /**
+     * Returns {@code true} if all default filters are disabled; {@code false} otherwise.
+     */
+    public abstract boolean disableToolDefaultReadFilters();
```

Comment:
I made them all abstract. My first thought was to make defaults the one with collections, and enforce the one for disabling because it is the only one affecting filters that are in the plugin and not in the actual collection. Anyway, it makes more sense to have all of them abstract....